x-pack/plugins/enterprise_search/public/applications/shared/layout/nav.tsx

```diff
@@ -18,13 +18,17 @@ import {
   ENTERPRISE_SEARCH_OVERVIEW_PLUGIN,
   WORKPLACE_SEARCH_PLUGIN,
 } from '../../../../common/constants';
+import { enableBehavioralAnalyticsSection } from '../../../../common/ui_settings_keys';
 import { SEARCH_INDICES_PATH } from '../../enterprise_search_content/routes';
 import { KibanaLogic } from '../kibana';
 
 import { generateNavLink } from './nav_link_helpers';
 
 export const useEnterpriseSearchNav = () => {
-  const { productAccess } = useValues(KibanaLogic);
+  const { productAccess, uiSettings } = useValues(KibanaLogic);
```

Comment:
FYI @TattdCodeMonkey - added uiSettings through KibanaLogic. This will help us with mocking.